src/DssSpell.sol

```diff
@@ -71,9 +101,58 @@ contract DssSpellAction is DssAction {
     uint256 internal constant SIX_PT_ONE_NINE_PCT_RATE    = 1000000001904482384730282575;
     uint256 internal constant FIVE_PT_FOUR_FOUR_PCT_RATE  = 1000000001679727448331902751;
 
+    // Operator address
+    address internal constant RWA015_A_OPERATOR          = 0x23a10f09Fac6CCDbfb6d9f0215C795F9591D7476;
+    // Custody address
+    address internal constant RWA015_A_CUSTODY           = 0x65729807485F6f7695AF863d97D62140B7d69d83;
+
+    address internal constant RWA015_A_OUTPUT_CONDUIT    = address(0); // TODO
```

Comment:
Could you maybe deploy current version of the conduit code in the meantime to be able to run the tests? You can keep `TODO` here and redeploy final version once the audit is complete